tutorrichie/templates/richie/apps/env.d/production

```diff
@@ -15,3 +15,20 @@ EDX_BASE_URL={% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }
 EDX_JS_BACKEND=openedx-hawthorn
 AUTHENTICATION_BASE_URL={% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}
 AUTHENTICATION_BACKEND=openedx-hawthorn
+CACHE_DEFAULT_LOCATION=redis://@redis:6379/3
+CACHE_DEFAULT_BACKEND=django_redis.cache.RedisCache
+CACHE_DEFAULT_OPTIONS={"CLIENT_CLASS": "django_redis.client.DefaultClient"}
+CACHE_DEFAULT_TIMEOUT=30
+AWS_DEFAULT_ACL=public-read
+DJANGO_AWS_ACCESS_KEY_ID={{ OPENEDX_AWS_ACCESS_KEY }}
+DJANGO_AWS_SECRET_ACCESS_KEY={{ OPENEDX_AWS_SECRET_ACCESS_KEY }}
+RICHIE_COURSE_RUN_SYNC_SECRETS=7uop42Ir0TKgQ0zVpVir
```

Comment:
Shouldn't this be a variable?

Reply:
It should. Everything in this file should be managed via varibles. Specially since this is a public repository.

Same for the `tutorrichie/plugin.py` file